spec/views/post/show.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts show view', type: :system do
+  describe 'Show page' do
+    subject { User.create(name: 'Ashley', photo: 'https://www.anisearch.de/images/character/cover/full/0/820.webp', bio: 'Solider', posts_counter: 0) }
+
+    it 'Display posts view content' do
+      post = Post.create(author_id: subject.id, title: 'Hello 1', text: 'This is my post', comments_counter: 0,
+                         likes_counter: 0)
+      Post.create(author_id: subject.id, title: 'Hello 2', text: 'This is my post 2', comments_counter: 0,
+                  likes_counter: 0)
+      Post.create(author_id: subject.id, title: 'Hello 3', text: 'This is my post 3', comments_counter: 0,
+                  likes_counter: 0)
+      comment = Comment.create(author_id: subject.id, post_id: post.id, text: 'Test comment')
+      visit user_posts_path(subject, post)
+
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(subject.name)
+      expect(page).to have_content('Comments: 1')
+      expect(page).to have_content('Likes: 0')
+      expect(page).to have_content(post.text)
+      expect(page).to have_content('Ashley: Test comment')
+      expect(page).to have_content(comment.text)
+    end
```

Comment:
- [ ] Kindly wrap each assertion within an "it" block as pointed out above and ensure all assertions for the post show page are present.